test/StakeManager.js

```diff
@@ -21,17 +30,23 @@ describe('StakeManager', function () {
     let blockManager;
     let parameters;
     let stakeManager;
+    let stakeRegulator;
```

Comment:
Add stakeRegulator to the end, the reason being it keeps the diff clean and prevents potential conflicts.

I see the reason for adding it here as it makes sense to add it along stakeManager. You can take a call with regards to my suggestion vs how it is right now.